diesel_cli/src/print_schema.rs

```diff
@@ -557,11 +666,32 @@ impl<'a> Display for ColumnDefinitions<'a> {
                     }
                 }
 
+                #[cfg(feature = "mysql")]
+                let mut column_type = column.ty.rust_name.clone();
+
+                #[cfg(not(feature = "mysql"))]
+                let column_type = column.ty.rust_name.clone();
+
+                #[cfg(feature = "mysql")]
+                {
+                    if let Some(custom_type) = self
+                        .custom_type_defs
+                        .types_sorted
+                        .iter()
+                        .find(|t| t.column_name == column.rust_name)
+                    {
+                        column_type = custom_type.generate_rust_name();
+                    }
+                }
+
+                #[cfg(not(feature = "mysql"))]
+                let _ = self.custom_type_defs;
+
                 if column.rust_name == column.sql_name {
-                    writeln!(out, "{} -> {},", column.sql_name, column.ty)?;
+                    writeln!(out, "{} -> {},", column.sql_name, column_type)?;
                 } else {
                     writeln!(out, r#"#[sql_name = "{}"]"#, column.sql_name)?;
-                    writeln!(out, "{} -> {},", column.rust_name, column.ty)?;
+                    writeln!(out, "{} -> {},", column.rust_name, column_type)?;
```

Comment:
This change is causing the CI failures. The old code was calling the `Display` implementation of the `ColumnType` type here: https://github.com/diesel-rs/diesel/blob/8d64a4011bed57851e3cd747df6d855d44ab9993/diesel_cli/src/infer_schema_internals/data_structures.rs#L74-L97

That generates some important wrapper-types depending on the nullability (and other conditions) for the type. By just using the the type name itself we loose that information. It's likely a better idea to just modify the stored values for `ColumnType::rust_name` at some stage for all custom mysql type fields and leave that code here unchanged